app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -1445,7 +1451,7 @@ public boolean canOpenNewWindow() {
     @Override
     public void openNewWindow(String uri) {
         WindowWidget newWindow = mWindows.addWindow();
-        if (newWindow != null) {
+        if (newWindow.getSession() != null) {
```

Comment:
Even though `Windows.addWindow()` is marked as `@NonNull` for the return type, it actually can return `null`. So probably need to both fix the `addWindow()` annotation as well as revert this  change and make it:
```
    if ((newWindow != null) && (newWindow.getSession() != null)) {
        ...
    }
```

Reply:
Fixed